### PR TITLE
BE-215: Added initial school prices

### PR DESCRIPTION
**Pull request by shangerdi**

### CODE REVIEW

File: server/app/models.py [hunk 1]
```diff
@@ -165,6 +165,48 @@ def get_photo_url_list(self):
     def get_member_service_list(self):
         return self.member_services.all().values_list('id', flat=True) or []
 
+    def init_prices(self):
```
Comment (1e0ng) [inline server/app/models.py:168]: It's better to add a unit test for this method.

Comment (shangerdi) [reply server/app/models.py:168]: yeah, and there will be a trouble if something wrong during the init, because it will not init again ...

Comment (shangerdi) [reply server/app/models.py:168]: I think we can go ahead and do some improvement later.

Comment (1e0ng) [reply server/app/models.py:168]: You may add a simple unit test, like just checking there exists a `priceconfig` after executing this method.

Comment (shangerdi) [reply server/app/models.py:168]: Added a simple one

**Comment by leonsim** [comment]
LGTM

**Comment by 1e0ng** [comment]
LGTM

**Pull request merged**